app/lib/fixAddress.js

```diff
@@ -0,0 +1,31 @@
+const fields = ['line1', 'line2', 'line3', 'city', 'county'];
```

Comment:
see comment above about reducing the number components and ensuring db-updater only performs db updates.